airgun/views/hostgroup.py

```diff
@@ -4,6 +4,7 @@
 from widgetastic.widget import TextInput
 from widgetastic.widget import View
 from widgetastic_patternfly import BreadCrumb
+from widgetastic_patternfly4 import Button as P4Button
```

Comment:
```suggestion
from widgetastic_patternfly4 import Button as PF4Button
```
same suggestion for line no. 23